### ISSUE TITLE
Support for multiple characters in a podcast

### ISSUE THREAD

**Issue by jtoy** [souzatharsis/podcastfy#200]
Many podcasts have quests or more than 2 people. Would be awesome to make the code support as many characters as supplied from the config files.

**Comment by souzatharsis** [comment]
Yes, I'm eager to Refactor the project to implement custom (arbitrary)
agents but time is lacking.
